core/opds2_import.py

```diff
@@ -514,14 +515,17 @@ def _extract_media_types_and_drm_scheme_from_link(
                     nested_acquisition = first_or_default(nested_acquisition.child)
                 [drm_type, media_type] = nested_types[-2:]
 
+                # We then check this returned pair of content types to make sure they match known
+                # book or audiobook and DRM types. If they do not match known types, then we skip
+                # the item.
                 if (
                     media_type in MediaTypes.BOOK_MEDIA_TYPES
                     or media_type in MediaTypes.AUDIOBOOK_MEDIA_TYPES
                 ) and drm_type in DeliveryMechanism.KNOWN_DRM_TYPES:
                     media_types_and_drm_scheme.append((media_type, drm_type))
 
         # There are no indirect links, then the link type points to the media, and
-        # there is no DRM on the item
+        # there is no DRM on the item.
```

Comment:
Same concern about the word `item` here.

Reply:
Updated both these comments to refer to link instead of item.